internal/service/rpz/record_rpz_cname_ipaddress_data_source_test.go

```diff
@@ -0,0 +1,121 @@
+package rpz_test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/infobloxopen/infoblox-nios-go-client/rpz"
+	"github.com/infobloxopen/terraform-provider-nios/internal/acctest"
+)
+
+func TestAccRecordRpzCnameIpaddressDataSource_Filters(t *testing.T) {
+	dataSourceName := "data.nios_rpz_record_rpz_cname_ipaddress.test"
+	resourceName := "nios_rpz_record_rpz_cname_ipaddress.test"
+	var v rpz.RecordRpzCnameIpaddress
+	rpZone := acctest.RandomNameWithPrefix("test-zone") + ".com"
+	nameIp := "11.0.0.30"
```

Comment:
```suggestion
	nameIP := "11.0.0.30"
```

applies for all occurences.